_locales/en_US/messages.json

```diff
@@ -415,7 +415,7 @@
 		"message": "Automatically add rules based on viewed DKIM signed e-mails"
 	},
 	"options_policy.signRules.autoAddRule.onlyIfFromAddressInSDID": {
-		"message": "Only if the From address is in the SDID"
+		"message": "Only if the From address is in the SDID:"
```

Comment:
Maybe that is not that clear form the GUI, but this checkbox and the drop down list following it in a newline are independent from each other.
So adding a `:` does not make sense here and would only be confusing.